CHANGELOG-FORK.md

```diff
@@ -4,6 +4,7 @@ Changes from v10.2.0
 - Feature:    
   - ADDED **HTTP API** `annotation/ways` in OSRM route response after `osrm-ranking` process(retrieve `annotation/ways` from `annotation/nodes`) [#296](https://github.com/Telenav/osrm-backend/pull/296)    
   - CHANGED for internal refactoring, moved `unidbpatch` and `mapsource` packages into `integration/util` folder [#300](https://github.com/Telenav/osrm-backend/pull/300)
+  - CHANGED for internal refactoring, refactor staiongraph to isolate algorithm, data structure and topological [#302]
```

Comment:
Would you please append the link so that the `#302` can be clickable? Meanwhile, this entry is only for the last PR, could you please also add one for this PR? Or maybe append also this PR's ID, i.e. one entry for both PR since this PR is the only supplement for the former one.      
Furthermore, `staiongraph` -> `stationgraph`?

Reply:
copy that :)